signer/src/emily_client.rs

```diff
@@ -82,6 +101,44 @@ impl EmilyInteract for EmilyClient {
         //     .await
         //     .map_err(EmilyClientError::GetDeposits)?;
     }
+
+    async fn update_broadcasted_deposits<'a>(
+        &'a self,
+        transaction: &'a UnsignedTransaction<'a>,
+        bitcoin_chain_tip: &'a BitcoinBlockRef,
+    ) -> Result<UpdateDepositsResponse, Error> {
+        let deposits = transaction.requests.iter().filter_map(|e| match e {
+            RequestRef::Deposit(d) => Some(d),
+            _ => None,
+        });
+
+        let mut update_request = Vec::new();
+        for deposit in deposits {
+            update_request.push(DepositUpdate {
+                bitcoin_tx_output_index: i32::try_from(deposit.outpoint.vout)
+                    .map_err(|_| Error::TypeConversion)?,
+                bitcoin_txid: deposit.outpoint.txid.encode_hex(),
+                fulfillment: None,
+                last_update_block_hash: bitcoin_chain_tip.block_hash.encode_hex(),
+                last_update_height: i64::try_from(bitcoin_chain_tip.block_height)
+                    .map_err(|_| Error::TypeConversion)?,
+                status: Status::Accepted,
+                status_message: "TODO?".to_string(),
```

Comment:
Should this `status_message` be something specific for the accepted case?

Reply:
I don't think so.

It's not clear if we should ever put anything here. If we did, I'm not sure why we'd use something other than an enum value here, (I mean, we'd transform it into a string first but still). I'd just put an empty string here always.